docs/contracts.rst

```diff
@@ -1406,24 +1406,27 @@ Using For
 
 The directive ``using A for B;`` can be used to attach library
 functions (from the library ``A``) to any type (``B``).
-These functions will receive the object they are called on
-as their first parameter (like the ``self`` variable in
-Python).
 
 The effect of ``using A for *;`` is that the functions from
-the library ``A`` are attached to any type.
+the library ``A`` are attached to *any* type.
 
-In both situations, all functions, even those where the
-type of the first parameter does not match the type of
-the object, are attached. The type is checked at the
+By including a module, its data types including library functions are
```

Comment:
What do you mean by "including a module"?

Reply:
Original wording (not mine). Personally I'd use 'library' instead of 'module', but did not want to mess with the original author's intent too much.

```
The using A for B; directive is active for the current scope, which is limited to a contract for now but will be lifted to the global scope later, so that by including a module, its data types including library functions are available without having to add further code.
```

Change to 'library'?